src/main/java/com/game/Battleship/models/Cell.java

```diff
@@ -2,25 +2,25 @@
 
 public class Cell {
 
-    private String type;
-    private String value = null;
+    private String type;//type could be: "water","Battleship", "Cruiser", "Submarine", "Destroyer"
+    private String value = null; //value could be null for initial state,"Ship"(when a ship is placed), "Hit"(when type is one of the ship), "Missed"(when type is water)
```

Comment:
Prefer to use enums for these string values.

https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Reply:
Done